spec/abstractops.html

```diff
@@ -1606,10 +1606,13 @@ <h1>
       <dd>It converts _argument_ to an integer representing its Number value with fractional part truncated (replacing *NaN* with 0), or throws a *RangeError* when that Number value is not finite.</dd>
     </dl>
     <emu-alg>
+      1. If _argument_ is *undefined*, throw a *RangeError* exception.
       1. Let _number_ be ? ToNumber(_argument_).
-      1. If _number_ is *NaN*, return 0.
-      1. If _number_ is *+&infin;*<sub>𝔽</sub> or *-&infin;*<sub>𝔽</sub>, throw a *RangeError* exception.
-      1. Return truncate(ℝ(number)).
+      1. If _number_ is *NaN*, -∞, or +∞, throw a *RangeError* exception.
+      1. If _number_ is *-0*<sub>𝔽</sub>, return 0.
+      1. Let _integer_ be floor(abs(ℝ(_number_))).
+      1. If _number_ < *-0*<sub>𝔽</sub>, set _integer_ to -_integer_.
+      1. Return _integer_.
```

Comment:
I think these 4 lines are equivalent to "Return truncate(ℝ(_number_))" which was already there, although I see I forgot the `_` around "number". [ToIntegerOrInfinity](https://tc39.es/ecma262/#sec-tointegerorinfinity) does this trick with floor(abs()) because truncate() didn't exist in the spec until recently.